CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fixed the catalog load button status after double clicking catalog files ([#2378](https://github.com/CARTAvis/carta-frontend/issues/2378)).
 * Synchronized the value format in the pan and zoom tab in the image view settings widget ([#2235](https://github.com/CARTAvis/carta-frontend/issues/2235)).
 * Fix save image/export regions bug which could cause directory overwrite or deletion ([#1377](https://github.com/CARTAvis/carta-backend/issues/1377)).
+* Fix incorrect rendering of image view when moving the window from Retina to non-Retina monitor, and vice versa ([[#2285](https://github.com/CARTAvis/carta-frontend/issues/2285)])
```

Comment:
Perhaps we can make it more general: `when moving the window to monitors with different screen resolution.`